troposphere/asg-elb-dns.py

```diff
@@ -1,14 +1,26 @@
 from troposphere import Join, Output, GetAtt
 from troposphere import Parameter, Ref, Template
-from troposphere.autoscaling import AutoScalingGroup, Tag
-from troposphere.policies import UpdatePolicy, AutoScalingRollingUpdate, CreationPolicy, AutoScalingCreationPolicy, ResourceSignal
+from troposphere.autoscaling import AutoScalingGroup, Tag, ScalingPolicy
+from troposphere.cloudwatch import Alarm, MetricDimension
+from troposphere.policies import UpdatePolicy, AutoScalingRollingUpdate, CreationPolicy, \
+    AutoScalingCreationPolicy, ResourceSignal
 import troposphere.elasticloadbalancing as elb
 from troposphere.route53 import RecordSetType
+import sys
 
 __author__ = 'Jose Armesto'
 
 
 def generate_cloudformation_template():
+    enable_elb = sys.argv[1]
+    input_scaling_policies = eval(sys.argv[2])
+    input_alarms = eval(sys.argv[3])
```

Comment:
Totally @joaoqalves ! The use case here was that I had a literal dictionary, in a string, like `"{ key: "value"}"` that I needed to convert to an actual dictionary in python. Is there a constructor I can use to avoid using `eval`?